nestkernel/source_table_position.h

```diff
@@ -97,15 +106,17 @@ SourceTablePosition::wrap_position( const std::vector< std::vector< BlockVector<
       continue;
     }
 
+    // if we can not find a valid lcid by decreasing synapse or thread
+    // indices, we have read all entries
     assert( tid < 0 );
     assert( syn_id < 0 );
     assert( lcid < 0 );
```

Comment:
I would tighten the last assert to `lcid == -1`, ensuring that `lcid + 1 >= 0` always.

Reply:
shouldn't we do this for all indices?